tools/docker/builder/openwrt/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh -e
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+rootdir="${scriptdir%/*/*/*/*}"
+
+if [ ! -f "$1" ] ; then
+    echo "Please specify a file containing the build args as the
+    first argument"
+    exit 1
+fi
```

Comment:
AFAICS there are now defaults for everything, so it is not actually needed to pass a config file.

However, I think it's nicer to use command line arguments instead of a config file, using getopt, like we do in most other scripts.

This is something we can easily update later so (nitpick).

Reply:
Indeed there are now defaults for everything. Now that you mention it, I should remove the default for TARGET and TARGET_PLATFORM_TYPE, as this build script will be used to build for other targets as well.
The idea of the config file was to make it easy for anyone to build for a specific target, without the need to know what TARGET and TARGET_PLATFORM_TYPE should be.

What I could do instead is to add a mandatory argument for the target and list the targets in the same script (currently only turris-omnia).
I'll rebase on master and keep it like that for now, please ping me if you want that done before merging.